src/components/member-profile/index.js

```diff
@@ -107,15 +107,79 @@ const Profile = (props) => {
       });
   };
 
+  const closeModal = () => {
+    setShowModal(false);
+    setTimeout(function () {
+      document.querySelector('#introButton').focus();
+      document.querySelector('body').style.overflowY = 'auto';
+    }, 50);
+  };
+
+  const handleGetIntroClick = () => {
+    setShowModal(true);
+    setTimeout(function () {
+      document
+        .querySelector('#modelContent')
+        .querySelectorAll('form input')[0]
+        .focus();
+      document.querySelector('body').style.overflowY = 'hidden';
+      document
+        .querySelector('#modelContent')
```

Comment:
Why can't we go with synthetic events here...
Try exploring more here 😃 

https://reactjs.org/docs/events.html#gatsby-focus-wrapper

Reply:
changed in the latest commit